ballista/client/src/context.rs

```diff
@@ -942,6 +958,11 @@ mod standalone_tests {
             "+--------------------+",
         ];
         assert_result_eq(expected, &res);
+    }
+    #[tokio::test]
+    #[ignore] // TODO fix this test - it never completes
```

Comment:
I will fix this as part of https://github.com/apache/datafusion-ballista/pull/1052